### ISSUE TITLE
ValueError at / Unsupported image shape: ()

### ISSUE THREAD

**Issue by AdarshPan** [stratospark/keras-multiprocess-image-data-generator#5]
I am using keras 2.4.3 with tensorflow 2.5.0

Please help,its urgent !

here is the code:

    if request.method == 'POST' and request.FILES['myfile']:
        # post = request.method == 'POST'
        myfile = request.FILES['myfile']#try using get
        #load_modelo()
        img = image.load_img(myfile, target_size=(64,64))
        img = image.img_to_array(img)
        img = np.expand_dims(img, axis=0)
        img = img/255
